app/views/movies/_movie_modal_innard.html.erb

```diff
@@ -7,7 +7,8 @@
 
         <% if movie.in_db && movie.lists.by_user(current_user).present? %>
 
-        <div class="partial-spacer">
+        <div class="partial-spacer ui-front">
+          <!-- ui-front is needed for jquery autocomplete in the modal -->
```

Comment:
The reason this is needed is explained in this SO post: https://stackoverflow.com/a/25021925. Basically, in the boostrap modal, this class needs to be defined on the parent of the element that will use jquery autocomplete.

Reply:
Dag. Nice work in figuring that out!